invites/async/reply.js

```diff
@@ -27,14 +27,13 @@ module.exports = function (server) {
 
     getInvite(reply.branch, (err, invite) => {
       if (err) return callback(err)
+      const { value: { content: { recps } } } = invite
       var whoami = server.whoami()
-      var notInvited = invite.recipient !== whoami.id
+      let recipients = recps.filter(recp => recp !== whoami.id)
+      var notInvited = recipients.length !== 1
```

Comment:
I think the logic of this should be : "check if I'm here in this invite" (where the question is currently "is there a person other than me in this invite" ... which I don't think is an accurate test of whether you were invited)

Reply:
:white_check_mark: done